homeassistant/components/triggercmd/__init__.py

```diff
@@ -0,0 +1,35 @@
+"""The TRIGGERcmd component."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from . import hub
+
+PLATFORMS: list[str] = ["switch"]
```

Comment:
Please use the Platform enum instead https://github.com/home-assistant/core/blob/dev/homeassistant/const.py#L39